### DISCUSSION TITLE
[Projects Beta] Edit Multiple Issues

### DISCUSSION THREAD

**Discussion by michaelfedell** [community/community#12356] [Projects and Issues]
It would be hugely useful to select/edit multiple issues from the Table View. We often create a large set of related issues, entering just a quick title for each, and then want to bulk-assign them to a member or assign them all to a particular category, etc.

**Comment by SpencerKaiser** [comment]
To add some extra context to this; this is particularly helpful when creating a project and migrating all your existing issues - the main reason I didn't really use it during the beta was because of the missing bulk add functionality. Now that everything is in my project, I'd LOVE to be able to just bulk set status or anything else to make things simple...

**Reply by smac711** [reply]
I'm also really feeling the pain of having to go through projects using multiple repos and not being able to bulk edit.

**Reply by austincondiff** [reply]
💥 Ouch! 🤕 So am I! 😭

**Comment by Luc45** [comment]
Another use-case: Transferring open issues from previous sprint to the current sprint. (eg: Move all `is:open sprint:@previous` to current sprint)

**Comment by kakkunpakkun** [comment]
Is there any way to sort multiple issues manually in the Table View? I really want to do that operation to manage priority after adding many issues to my project.

**Comment by Rvervuurt** [comment]
The weird thing is, that you can select multiple issues at once, but after that, no functionality is added. One can't even drag multiple issues at once?

**Comment by NickiCoates** [comment]
Bulk edit options would be really helpful for us too

**Comment by armenzg** [comment]
FYI when looking at the Kanban view I managed to select all issues in one column and drag them to another one. Yay!

Moving between sprints is a very basic functionality for anyone running sprints.

A very easy way to solve this is by giving the right click a bunch of options that could be applied to the selected issues. See screenshot if I'm not clear.
<img width="341" alt="image" src="https://github.com/community/community/assets/44410/29a642ad-f635-49d7-9fa3-afd741434d6c">

**Comment by MKlimuszka** [comment]
Definitely need the ability to do this as a project manager. See functionality in [JIRA ](https://confluence.atlassian.com/jirasoftwareserver/editing-multiple-issues-at-the-same-time-939938937.html)and [Zube ](https://zube.io/blog/introducing-the-issue-manager/) for examples.

**Comment by mtcA6** [comment]
The ability to drag the assignee or the labels across multiple stories is already sort of there and i have no issues w/updating some of the project metadata on a bunch of tickets at once since the project operates like a spreadsheet in many ways. 

Where things get annoying is if I want to drag a bunch of items around to show the priority of the items. Sure I can use the priority field BUT then within a list of high priority items some have to come before others. 

I don't even use the kanban board because the multiple update functionality is so crummy.

**Comment by DarioMetalcam** [comment]
+1 on sprint change
+1 on reassigning tickets to someone else (because person was moved to another project)

**Comment by pgorod** [comment]
I just need to be able to add a label to many Issues at once.
